clo-workflows/Scripts/src/service/dataService/SpDataService.ts

```diff
@@ -204,9 +233,9 @@ export class CloRequestElementParser extends ODataDefaultParser {
         const parsedResponse = await super.parse(response)
 
         // the parsedResponse may be an array of response objects or a single object, depending on what was requested
-        if(Array.isArray(parsedResponse)) {
+        if (Array.isArray(parsedResponse)) {
```

Comment:
is prettier making these changes? the rest of the code has no space in if(...) statements so these should be undone